data/ShivangiS04.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Shivangi Singh",
+  "type": "Personal",
+  "bio": "Just an Engineering undergrad trying to find her place in the world.",
+
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aloobandar01",
```

Comment:
You have 2 different twitter url on line 9 & 14. Please, check which one is correct and use it here.

Reply:
The fix can come in another PR. Thanks for the review